bpfman/src/serve.rs

```diff
@@ -117,12 +126,25 @@ pub async fn serve(
     Ok(())
 }
 
-pub(crate) async fn shutdown_handler() {
+pub(crate) async fn shutdown_handler(use_activity_timer: bool) {
```

Comment:
Why not just have two methods `shutdown_handler_timer()` `shutdown_handler()` rather than all the stuff with `&mut bool` which is kind of a weird way to send signals between threads